test/Microsoft.Azure.ServiceBus.UnitTests/QueueSessionTests.cs

```diff
@@ -188,6 +189,28 @@ public sealed class QueueSessionTests
             }
         }
 
+        [Theory]
+        [MemberData(nameof(TestPermutations))]
+        [DisplayTestMethodName]
+        async Task ServerWaitTimeoutTest(string queueName, int messageCount = 1)
```

Comment:
What `ServerWaitTimeoutTest` is doing? Don't force to read the whole test just to decipher what the outcome should be. You don't do that with regular methods, right? 😉